services/api-server/src/simcore_service_api_server/core/application.py

```diff
@@ -9,6 +9,10 @@
     setup_prometheus_instrumentation,
 )
 from servicelib.logging_utils import config_all_loggers
+from simcore_service_api_server.api.errors.custom_errors import (
```

Comment:
should be relative import, actually all of them.
This will probably also fix issues between ruff autosorting and isort

Reply:
Good point. Thanks, I will fix